02_bubble/brvishnu92/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+
+	swap := true
+	for swap {
+		z := s[0]
+		swap = false
+		for i, x := range s {
+			if x < z && i != 0 {
+				swap = true
+				temp := s[i-1]
+				s[i-1], s[i] = s[i], temp
```

Comment:
You probably don't need the temp variable here.

You can write `s[i-1], s[i] = s[i], s[i-1]` and it will swap the values, the right hand side is evaluated prior to the assignment of those values.

Reply:
Oh yeah, thanks for that @nicholascross 👍